service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ActionCaseManagementOrderControllerAboutToSubmitTest.java

```diff
@@ -68,6 +72,7 @@
     private static final UUID ID = randomUUID();
 
     private final DocumentReference cmoDocument = buildFromDocument(document());
+    private CaseDetails populatedCaseDetails;
```

Comment:
this could be private final CaseDetails populatedCaseDetails  = populatedCaseDetails();

and removed from setup

Reply:
I update populatedCaseDetails in each test, so needs to be reset in the beforeEach

Reply:
I update populatedCaseDetails directly so I need to reset before each test in the before each